src/property-filter/property-filter-autosuggest.tsx

```diff
@@ -34,7 +34,9 @@ import { joinStrings } from '../internal/utils/strings';
 const DROPDOWN_WIDTH_OPTIONS_LIST = 300;
 const DROPDOWN_WIDTH_CUSTOM_FORM = 200;
 
-export interface PropertyFilterAutosuggestProps extends AutosuggestProps, InternalBaseComponentProps {
+export interface PropertyFilterAutosuggestProps
+  extends Omit<AutosuggestProps, 'filteringResultsText'>,
```

Comment:
So we don't want to implement this for Property filter? Why?

Reply:
We decided that Property Filter and Attribute editor will be out of scope. We didn't receive requests for them, so if there will be some in the future, we can do a follow-up.

One more thing is that Property filter needs more investigation from a design perspective and it's more tricky to come up with meaningful writing guidelines

Reply:
Cool, works for me!